pages/reference/supervisor/bandwidth-reduction.md

```diff
@@ -4,52 +4,41 @@ title: Reduce bandwidth usage
+balenaOS and the device Supervisor provide convenient features that make it easy to manage a device remotely, like forwarding container logs, hardware metrics reporting, and Cloudlink for on-demand external access. However, these features consume network bandwidth, and typically cellular connectivity is charged by the megabyte. To allow you to trade off between cost and features, you may configure their use individually.
 
-In order to give our power users control over data flow, we enabled a few `BALENA_` variables in the [configuration][configuration] tab.
+Most of these controls are available from the fleet/device configuration tabs on the dashboard as well as via Supervisor variables. The NetworkManager connectivity check may be configured via `config.json`.
 
-| Variable                               | Allowed Value      | Action                                             | Default | Available with Supervisor Version |
-|----------------------------------------|--------------------|----------------------------------------------------|---------|-----------------------------------|
-| `BALENA_SUPERVISOR_VPN_CONTROL`        | true/false         | Enable / Disable cloudlink                         | true    | v1.1.0                            |
-| `BALENA_SUPERVISOR_CONNECTIVITY_CHECK` | true/false         | Enable / Disable cloudlink connectivity check      | true    | v1.3.0                            |
-| `BALENA_SUPERVISOR_POLL_INTERVAL`      | 600000 to 86400000 | API Poll interval in milliseconds                  | 900000  | v1.3.0                            |
-| `BALENA_SUPERVISOR_LOG_CONTROL`        | true/false         | Enable / Disable logs from being sent to API       | true    | v1.3.0                            |
-| `BALENA_SUPERVISOR_HARDWARE_METRICS`   | true/false         | Enable / Disable device hardware metrics reporting | true    | v12.8.0                           |
+**The minimum bandwidth required is approximately 1 MB per month**, when turning off all of the features below and setting API Polling to the minimum value of once per day. These measurements were made with a Quectel modem on a SixFab HAT on a Raspberry Pi4, and using a Twilio North American SIM. See this [blog post](https://www.balena.io/blog/cellular-iot-isnt-as-hard-as-you-think/) for a similar setup. The device was running balenaOS v2.107.10 and Supervisor v14.4.4.
 
-Side Effects / Warnings
--------------------
+| Control | Monthly Bandwidth if enabled | Variable Name<br>`BALENA_SUPERVISOR...` | Comments |
```

Comment:
```suggestion
| Control | Monthly Bandwidth if enabled | Variable Name| Comments |
```

Much similar to write the complete variable name @kb2ma because someone would be copying these straightup and not liking the experience.

Reply:
Suggestion: replace `Comments` with `Use` or `description`

Reply:
I was trying to conserve horizontal space, but I like the practicality of your suggestion to include the full name. I'm going to try putting the variable name below the name of the feature in the leftmost cell.